bin/templates/cordova/lib/build.js

```diff
@@ -542,5 +653,12 @@ module.exports.help = function() {
     console.log('    \'--versionCode=#\': Override versionCode for this build. Useful for uploading multiple APKs. Requires --gradle.');
     console.log('    \'--minSdkVersion=#\': Override minSdkVersion for this build. Useful for uploading multiple APKs. Requires --gradle.');
     console.log('    \'--gradleArg=<gradle command line arg>\': Extra args to pass to the gradle command. Use one flag per arg. Ex. --gradleArg=-PcdvBuildMultipleApks=true');
+    console.log('');
+    console.log('Signed APK flags:');
```

Comment:
Would be good to mention that these are overrides for what's already in `debug-signing.properties` / `release-signing.properties`